### DISCUSSION TITLE
How do I script LVM on top of LUKS?

### DISCUSSION THREAD

**Discussion by mattyspangler** [archlinux/archinstall#1819] [Q&A] [answered]
I'm looking to script LVM on top of LUKS with my arch installs. What would be the best way to go about this? Should I make a plugin, use an existing function, or are you guys planning to support this feature?

**Comment by Torxed** [comment, marked as answer]
It's a planned feature, and it should'nt be too long now :)

**Reply by svartkanin** [reply]
@Torxed have you started on this? Otherwise this would be the next item on my list

**Reply by OliverGeneser** [reply]
@svartkanin Any updates? 😊

**Reply by svartkanin** [reply]
I have started working on it but there's a lot of things to consider
- manual config
- default config
- encryption 
etc 

It'll take a bit until it's ready, i may create a WIP PR when I have the menu ready to collect some feedback. Don't wanba put an ETA on it but maybe in the next weeks

**Reply by OliverGeneser** [reply]
@svartkanin Okay :smile: Please let me know if there is anything I can do to help :+1:

**Reply by LordRatte** [reply]
What is the progress on this? How can we get involved?

**Reply by svartkanin** [reply]
WIP PR is here https://github.com/archlinux/archinstall/pull/2104
Focus in the past weeks/months was primarily to get the new partitioning working and resolve bugs and issues for it. 
The LVM work will be picked up again now

**Reply by svartkanin** [reply]
The PR is now ready and can be tested https://github.com/archlinux/archinstall/pull/2104
